bme_test_suite.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- hosts: deploy_node
+  name: Install and run tests
+  vars:
+    tempest_root: /opt/tempest_untagged
+    test_name: persistent-resources-tests
```

Comment:
These were set as default while testing these playbooks, no real decision on that other than making sure tempest would run not influenced by a default test name/type. Tempest would be a better default.